downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -58,6 +58,17 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### New features
 
+#### `data.jsondata` packages
+
+- Introduced constraint validation support.
+- Introduced support for parsing JSON with union types as expected types.
+
+#### `data.xmldata` packages
+
+- Introduced constraint validation support.
```

Comment:
```suggestion
- Introduced validate the output results using `ballerina/constraint` module. wdyt?.
```

Reply:
Im suggesting something bit more descxriptive than the general senetence

Reply:
@MaryamZi  WDYT?

Reply:
```suggestion
- Introduced constraint validation support, allowing validation of the output against constraints specified in the target type.
```